README.md

```diff
@@ -63,7 +72,8 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
+Scope encompasses all the parent functions and means that an inner function will be influenced by it's parent. Children functions are not included in the scope and cannot be referenced. You can call variables and functions that are within the scope, but anything that is outside the scope cannot be referenced.
```

Comment:
Yes, good answers but I wanted to point out one rule that you did not explicitly mention: every function and only functions create a new local scope. Additionally, scope does not just refer to functions, it includes variables and objects that you have access too as well.